fab_irma.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import datetime
+import time
+import logging
+
+import qrcode
+import requests
+import subprocess
+
+RELYING_PARTY_ID = ""
+RELYING_PARTY_NAME = ""
+RELYING_PARTY_LOGO = ""
+API_KEY = ""
+IRMA_TOKEN = "secret-fake-token"
+IRMA_SERVER = "https://pnonvsmdy9.execute-api.ap-southeast-2.amazonaws.com/dev"
```

Comment:
Would not be better to get IRMA_SERVER_URL from env file and not expose ours?

Reply:
Wanted to remove but forgot! thanks for reminding